library/core/src/intrinsics.rs

```diff
@@ -2578,8 +2578,7 @@ extern "rust-intrinsic" {
 ///
 /// Currently such an assertion would always succeed; until Rust decides
 /// otherwise, that principle should not be violated.
-#[rustc_const_unstable(feature = "const_eval_select", issue = "none")]
-#[unstable(feature = "core_intrinsics", issue = "none")]
```

Comment:
don't we need the non-const stability too?

Reply:
Everything in this module is implicitly unstable under core_intrinsics, I think. Most of the other intrinsics here do not have that attribute either, no idea what it is present for some of them.